TROUBLESHOOTING.md

```diff
@@ -119,9 +119,10 @@ For environments with multiple root workspaces, place the `.vscode/settings.json
 When filing an issue, make sure you do the following:
 
 -   Check existing issues for the same problem (also see the "Known Issues" section above for widespread problems).
--   Enable trace logging by adding `"python.analysis.logLevel": "Trace"` to your settings.json configuration file.
+-   Enable trace logging by adding `"python.analysis.logLevel": "Trace"` to your settings.json configuration file or by using the `Pylance: Start Logging` command.
     -   Adding this will cause a large amount of info to be printed to the Python output panel.
         This should not be left long term, as the performance impact of the logging is significant.
+        Use `Pylance: Stop Logging` to turn off the additional info.
```

Comment:
Maybe rather than "turn off the additional info", "disable trace logging" would be more self-explanatory since that's what we called it earlier.
```suggestion
        Use `Pylance: Stop Logging` to disable trace logging when you are done.
```